cmd/wigPeaks/wigPeaks.go

```diff
@@ -0,0 +1,84 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/vertgenlab/gonomics/bed"
+	"github.com/vertgenlab/gonomics/fileio"
+	"github.com/vertgenlab/gonomics/wig"
+	"log"
+)
+
+func wigPeaks(in_wig string, out_bed string, threshold float64) { //threshold is float64 because WigValue Value aka v2.Value is float64. Need to cast int64 for bed.Bed for now
+
+	//check Go documentation for gonomics > wig > functions written for wig
+	records := wig.Read(in_wig) //type is []*Wig, aka slice of pointers to Wig
+	inPeak := false             //a bool to keep track of whether we are inside a peak
+	//var answer []*bed.Bed = make([]*bed.Bed,0) //to store list of peaks as slice of beds, use pointer to pass by reference and edit slice of beds, "make" command needs legnth argument so start with 0
+	var current *bed.Bed              //to store the current peak as a bed entry
+	out := fileio.EasyCreate(out_bed) //instead of answer, use "chan" method to write as we go. out has type EasyCreate defined as a gonomics struct, one of the fields is File which has type *os.File, needed for bed.WriteBed
+	defer out.Close()
+
+	for _, v1 := range records { //in range for loop, i is index (0,1,2..) which we don't use in this instance, v is value (content of slice). Record is slice of wig struct, each iteration is slice/object of wig struct, which looks like a block and is often organized by chromosomes (or part of chromosome), and each wig struct will produce independent peaks. The v1 iteration has chr=chrom3,step=100, and a list of WigValues where there are positions+values, values like 11 22 100 etc.
```

Comment:
These comments are really wonderful.  I would think about line wrapping them in the future so that it is easier to see the whole line at once (as a multi-line comment).